### PR TITLE
implement validate contact form.The content of the email field has to…

### PR DESCRIPTION
**Pull request by fithamlak**
=>implement validate contact form
  -The content of the email field has to be in lower case
  -When the user submits the form, the email validation function check if the email is in lower case
     - If the validation is OK, the form is sent.
     - If the validation is not OK the function show an error message to the user near the submit button informing them of the 
     error and the form is not sent.

please consider I have not finished "details-popup-windo" yet. date 12/23/2022

coding-partner-challenge

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,3 +27,18 @@ function closeNav() {
...
+  if (email !== email.toLowerCase()) {
+    event.preventDefault();
+    const errMessage = document.querySelector('.validation-message');
+    errMessage.innerText = error;
+  }
+}
+const form = document.getElementById('form');
+form.addEventListener('submit', (event) => {
+  const invalidEmail = 'Error, please consider to write all lower case letters.';
+  const email = form.elements.mail;
```
Comment (HAMMAS-SALEEM) [inline index.js:41]: Capture1
**[Optional]**

- [ ] It's great that you have implemented the validation 😊 but your message is not meaningful enough means you are not telling the user where to use all the lower case letters means in which text field. So you can say Please consider to write all lower case letters for email 👍

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @fithamlak 👋

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Follow Github Flow
✔️ Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop-styles.css [hunk 2]
```diff
@@ -504,3 +504,11 @@ button {
   border-radius: 8px;
   border: none;
 }
+
```
Comment (HAMMAS-SALEEM) [inline desktop-styles.css:507]: - [ ] Your summary is not descriptive enough. You didn't added any details in your summary that what are you doing in this pull request. Kindly add some details what are you doing in this pull request :+1:

Comment (HAMMAS-SALEEM) [inline desktop-styles.css:507]: - [ ] You didn't add linters for javascript which is ESLint. Kindly add the write set of linters 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submissions 🔴

## Hey @fithamlak :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements but i cannot proceed review on your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop-styles.css (hunk 2 above)
Comment (fithamlak) [reply desktop-styles.css:507]: There is already ESLint configuration please

**Review by emmiiorji — APPROVED** [review]
Hi @fithamlak, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

A little bit more

## Good Highlights

- Linter checks are passing
- Adhered to GitHub flow
- Validation works in contact form

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fithamlak** [comment]
Merge-validate-contact-form

**Pull request merged**